packages/core-database/src/repositories/blocks-business-repository.ts

```diff
@@ -0,0 +1,50 @@
+import { Database } from "@arkecosystem/core-interfaces";
+import { SearchParameterConverter } from "./utils/search-parameter-converter";
+
+export class BlocksBusinessRepository implements Database.IBlocksBusinessRepository {
+
+    constructor(private databaseServiceProvider: () => Database.IDatabaseService) {
+    }
+
+    public async findAll(params: any) {
+        /* In core-api/blocks repo. 'findAll' differs from search in that, it assumes 'equal' op for all passed in parameters
```

Comment:
The `eq` behaviour is how v1 behaved, that is why the `search` and `findAll` methods currently differ as `search` is exposed through the v2 API so we have to keep the behaviour of `findAll` as is until we drop the v1 API.

Reply:
OK, are we still using v1 pathways still? I wasn't too sure if the v2 migration meant deprecation of v1 code...

Reply:
All v1 or legacy code will be removed in 2.5 which will be roughly around summer this year, sadly have to keep it until then.